geopandas/base.py

```diff
@@ -149,8 +149,23 @@ def crs(self, value):
 
     @property
     def geom_type(self):
-        """Returns a ``Series`` of strings specifying the `Geometry Type` of each
-        object."""
+        """
+        Returns a ``Series`` of strings specifying the `Geometry Type` of each
+        object.
+
+        Examples
+        --------
+        >>> from shapely.geometry import Point, Polygon, LineString
+        >>> d = {'geometry': [Point(), Point(2,1),  Polygon([(0, 0), (1, 1), (1, 0)]),
```

Comment:
Can you leave out `Point()` for this example? It is unnecessarily confusing that it changes to empty GeometryCollection. That is not the point of this example.